src/.eslintrc

```diff
@@ -4,6 +4,7 @@
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
     "react/forbid-prop-types": "disabled",
     "comma-dangle": ["error", "never"],
-    "max-lines-per-function": ["error", 100]
+    "max-lines-per-function": ["error", 100],
+    "no-trailing-spaces": [2, { "skipBlankLines": true }]
```

Comment:
This will ignore the trailing spaces rule for blank lines. The rule still applies to trailing spaces in lines of code. Per the docs: ```These whitespace differences can be picked up by source control systems and flagged as diffs, causing frustration for developers.```

Reply:
👍